vm/systemSmartContracts/auction.go

```diff
@@ -497,22 +547,31 @@ func (s *stakingAuctionSC) isFunctionEnabled(nonce uint64) bool {
 func (s *stakingAuctionSC) stake(args *vmcommon.ContractCallInput) vmcommon.ReturnCode {
 	err := s.eei.UseGas(s.gasCost.MetaChainSystemSCsCost.Stake)
 	if err != nil {
+		s.eei.AddReturnMessage("insufficient gas")
```

Comment:
"insufficient gas limit" to use the same message as you return above

Reply:
done.